com.unity.multiplayer.mlapi/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -533,7 +536,14 @@ private bool ImportReferences(ModuleDefinition moduleDefinition)
         private void ProcessNetworkBehaviour(TypeDefinition typeDefinition)
```

Comment:
minor: following the similar pattern as `RuntimeAccessModifiersILPP`, we might just pass `string[] assemblyDefines` around instead of pinning `m_CompiledAssembly` to a local variable.